data/monitoring/prom-rule.yaml

```diff
@@ -18,3 +18,10 @@ spec:
           for: 5m
           labels:
             severity: critical
+        - alert: CnaoNotReady
+          annotations:
+            summary: one or more of CNAO deployed components is not ready.
+          expr: kubevirt_cnao_num_up_operators == 0
```

Comment:
same expr as above, maybe you mean kubevirt_cnao_ready ?

Reply:
the alert should fire when CNAO is not ready, so the name is fitting imo
but you are right that I used the wrong `expr` there. fixed. thanks!